src/services/config/service.ts

```diff
@@ -68,6 +68,12 @@ export const makeConfig = ({
       optional(() => bool(env.FORCE_DENCUN_FORK_MODE)) ?? false,
 
     CAPELLA_FORK_VERSION: optional(() => str(env.CAPELLA_FORK_VERSION)),
+
+    OPERATOR_IDENTIFIERS: json_arr(
+      env.ORACLE_ADDRESSES_ALLOWLIST,
+      (oracles) => oracles.map(num),
+      'Please, setup OPERATOR_IDENTIFIERS. Example: [1,2,3]'
```

Comment:
Why `ORACLE_ADDRESSES_ALLOWLIST` provided here?

Reply:
I saw it was removed in next PR. Let's remove this code from this PR

Reply:
It will be removed at the merge. We should not change anything now, so that there is no conflict. This PR is only needed to make it easier to read the main one

Reply:
There should no be merge conflicts if we just remove it from PR.
If we won't remove it, than we should merge second PR first

Reply:
As I mentioned above, this pull request was created for convenience. This code has already been fixed in another pull request, and I’ll merge them once the review is finished.